app/services/provider_data_api_client.rb

```diff
@@ -8,12 +8,20 @@ def contract_active?(office_code)
       )
     end
 
-    def user_office_details(user_login)
-      query(
-        "provider-users/#{ERB::Util.url_encode(user_login)}/provider-offices",
-        200 => ->(data) { data['officeCodes'] },
-        204 => [],
-      )
+    def user_office_details(user_login, api_version = nil)
```

Comment:
We're only ever going to want the new endpoint now

Reply:
true, but I think we should have versioning at least until we have confirmed the new version works properly and is stable. That said, I agree there's maybe not much value in retaining the logic for every api version ever though (e.g. when it moves to v2, v3, v4 etc it's very very unlikely we would ever have a scenario where we revert down by 2 versions), so will remove this argument in favour of just having the feature flag